### ISSUE TITLE
Blurry results

### ISSUE THREAD

**Issue by wuhaozhe** [naoto0804/pytorch-inpainting-with-partial-conv#55]
Hi, thanks for your project.
I've trained your model on a face dataset.
After 40K iterations, the result is still blurry.
![image](https://user-images.githubusercontent.com/18577194/143418223-d4a39b49-afe0-41ba-9cd3-e5126a1f1577.png)

Here is the loss log
![image](https://user-images.githubusercontent.com/18577194/143418408-258bf1f2-0d4f-48b7-b473-bff70ce3d3c4.png)
![image](https://user-images.githubusercontent.com/18577194/143418462-947c597f-8a8b-400c-ab6f-6529030c8efd.png)

I'm wondering is that normal? How long will it take to get good results?

**Comment by naoto0804** [comment]
I'm not sure how the progress goes in your faces dataset (in Places2, 160k is at least OK).
Since this implementation is based on the paper published four years ago, you might try recent methods if you want to get better results faster.

**Comment by husheng-maker** [comment]
Hi, I want to train my dataset use this model, But this code is about the Pales2 dataset, so how do I train my own dataset?

**Comment by naoto0804** [comment]
You can get your own dataset and modify places2.py to load the images in it.
